.upstream-tests/utils/nvidia/linux/perform_tests.bash

```diff
@@ -84,6 +84,11 @@ function report_and_exit {
   # |  Failed             : 2
   # |  Unexpectedly Passed: 1
 
+  if [ -e "${JSON_OUTPUT_PATH}" ]
```

Comment:
We're skipping the scoring here because DVS-SC will process the JSON log from `lit` correct to evaluate a score?

Reply:
Yes. VRL would calculate the score by checking the pattern "&&&&PASS/<OTHER state>"

Reply:
by the way, I see an unexpected compilation build failure in http://scdvsdsk.nvidia.com/jobs/8156/8156440/test_results.log
but the test result is pass because the final result is an expected failure.
I thought that this "expected failure" means an expected test failure but not expected build failure, isn't it?
if yes, I thought that we may need to file a bug.

Reply:
You can search the pattern "Compilation failed unexpectedly!" after you opened the above link

Reply:
This process may not be necessary. 
The original evaluation process would not have effect on the VRL score.

Reply:
Hi Wesley,

    I removed this logic just now. (I wanted to restrain all output that are printed in standard output stream, so I tried to remove the statistics if JSON_OUTPUT_PATH was defined)
    Please help to review other parts. Thanks!